src/localizationApi/LocalizationAPI.ts

```diff
@@ -33,6 +41,7 @@ export interface Locale {
 export interface LocalizationApiOptions {
```

Comment:
@jbevis let's update this api to have this interface/type, and also create an `const` alias called `i18nClient` and export it as well:

```
export interface LocalizationApiOptions {
  locales: LocaleList;
  defaultLocale?: Locale;
  imgMap?: ImgMap
}

export class LocalizationApi {
  constructor(private options: LocalizationApiOptions) {}

  ...
}
```

Reply:
@jbevis @logan-anderson we also probably want to "soft deprecate" `LocalizationApi` as the nom de jour for API classes are `{{ THING }}Client`, e.g:

```
apis: {
  github: new GithubClient(...),
  contentful: new ContentfulClient(...),
  i18n: new i18nClient(...)
}
```

Reply:
@chrisdmacrae @logan-anderson 

Whomever has some time, I can't seem to get the requested changes to work. Errors out on a number of rather obscure (to me) typescript errors. Would appreciate a second set of eyes.

```Type 'Locale | undefined' is not assignable to type 'Locale'.
  Type 'undefined' is not assignable to type 'Locale'```

Reply:
Updated the API with the requested changes.